package.json

```diff
@@ -81,7 +81,8 @@
     "build:libs": "yarn compile:umd && yarn compile:esm",
     "compile:umd": "tsc -p ./tsconfig.umd.json",
     "compile:esm": "tsc -p ./tsconfig.esm.json",
-    "check-dependencies": "node ./scripts/check-dependencies.js"
+    "check-dependencies": "node ./scripts/check-dependencies.js",
+    "pull-remote": "git pull https://github.com/redhat-developer/yaml-language-server.git main"
```

Comment:
this change shouldn't be there. 
I can remove this if you don't want this in scrips...

but this command is useful to merge changes from your repo into forked one...

Reply:
It is better to remove. It will confuse me :)